### ISSUE TITLE
cannot load github.com/gobuffalo/buffalo-pop/pop/v2/popmw

### ISSUE THREAD

**Issue by Pulgafree** [gobuffalo/buffalo#1959]
### Description

There is a misspelled package when generating a new project.

### Steps to Reproduce the Problem

  1. buffalo new api --api --db-type=cockroach
  2. buffalo dev

### Expected Behavior

Get in the console log:
`Starting application at http://127.0.0.1:3000`

### Actual Behavior

Got:
`cannot load github.com/gobuffalo/buffalo-pop/pop/v2/popmw: module github.com/gobuffalo/buffalo-pop/pop@latest found (v0.0.0-20200117125321-63e4d71afb90), but does not contain package github.com/gobuffalo/buffalo-pop/pop/v2/popmw`

### Info

<details>

```
Go: Checking installation
←[32m✓ The `go` executable was found on your system at: C:\Go\bin\go.exe←[0m

-> Go: Checking minimum version requirements
←[32m✓ Your version of Go, 1.13.10, meets the minimum requirements.←[0m

-> Go: Checking Package Management
←[32m✓ You are using Go Modules (`go`) for package management.←[0m

-> Go: Checking PATH
←[32m✓ Your PATH contains C:\Users\Isacar\go/bin.←[0m

-> Node: Checking installation
←[32m✓ The `node` executable was found on your system at: C:\Users\Isacar\scoop\apps\nodejs\current\node.exe←[0m

-> Node: Checking minimum version requirements
←[32m✓ Your version of Node, v13.12.0, meets the minimum requirements.←[0m

-> NPM: Checking installation
←[32m✓ The `npm` executable was found on your system at: C:\Users\Isacar\scoop\apps\nodejs\current\npm.cmd←[0m

-> NPM: Checking minimum version requirements
←[32m✓ Your version of NPM, 6.14.4, meets the minimum requirements.←[0m

-> Yarn: Checking installation
←[32m✓ The `yarnpkg` executable was found on your system at: C:\Program Files (x86)\Yarn\bin\yarnpkg.cmd←[0m

-> Yarn: Checking minimum version requirements
←[32m✓ Your version of Yarn, 1.22.0, meets the minimum requirements.←[0m

-> PostgreSQL: Checking installation
←[31m✘ The `postgres` executable could not be found on your system.←[0m
For help setting up your Postgres environment please follow the instructions for you platform at:

https://www.postgresql.org/download/

-> MySQL: Checking installation
←[31m✘ The `mysql` executable could not be found on your system.←[0m
For help setting up your MySQL environment please follow the instructions for you platform at:

https://www.mysql.com/downloads/

-> SQLite3: Checking installation
←[31m✘ The `sqlite3` executable could not be found on your system.←[0m
For help setting up your SQLite3 environment please follow the instructions for you platform at:

https://www.sqlite.org/download.html

-> Cockroach: Checking installation
←[32m✓ The `cockroach` executable was found on your system at: E:\cockroach.exe←[0m

-> Cockroach: Checking minimum version requirements
←[32m✓ Your version of Cockroach, 19.2.5, meets the minimum requirements.←[0m

-> Buffalo (CLI): Checking installation
←[32m✓ The `buffalo` executable was found on your system at: C:\Users\Isacar\scoop\shims\buffalo.exe←[0m

-> Buffalo (CLI): Checking minimum version requirements
←[32m✓ Your version of Buffalo (CLI), v0.16.3, meets the minimum requirements.←[0m

-> Buffalo: Application Details
Pwd         C:\Users\Isacar\go\src\aware.vasiko.com.ar\api
Root        C:\Users\Isacar\go\src\aware.vasiko.com.ar\api
GoPath      C:\Users\Isacar\go
PackagePkg  aware.vasiko.com.ar/api
ActionsPkg  aware.vasiko.com.ar/api/actions
ModelsPkg   aware.vasiko.com.ar/api/models
GriftsPkg   aware.vasiko.com.ar/api/grifts
WithModules true
Name        api
Bin         bin\api
VCS         git
WithPop     true
WithSQLite  false
WithDep     false
WithWebpack false
WithNodeJs  false
WithYarn    false
WithDocker  true
WithGrifts  true
AsWeb       false
AsAPI       true
InApp       true
PackageJSON {map[]}

-> Buffalo: config/buffalo-app.toml
name = "api"
bin = "bin\\api"
vcs = "git"
with_pop = true
with_sqlite = false
with_dep = false
with_webpack = false
with_nodejs = false
with_yarn = false
with_docker = true
with_grifts = true
as_web = false
as_api = true

-> Buffalo: config/buffalo-plugins.toml
[[plugin]]
  binary = "buffalo-pop"
  go_get = "github.com/gobuffalo/buffalo-pop/v2"

-> Buffalo: go.mod
module aware.vasiko.com.ar/api

go 1.13

require (
        github.com/gobuffalo/buffalo v0.15.5
        github.com/gobuffalo/buffalo-pop/v2 v2.0.5
        github.com/gobuffalo/envy v1.9.0
        github.com/gobuffalo/mw-contenttype v0.0.0-20190129203934-2554e742333b
        github.com/gobuffalo/mw-forcessl v0.0.0-20180802152810-73921ae7a130
        github.com/gobuffalo/mw-paramlogger v0.0.0-20190129202837-395da1998525
        github.com/gobuffalo/packr/v2 v2.8.0
        github.com/gobuffalo/pop/v5 v5.0.11
        github.com/gobuffalo/suite v2.8.2+incompatible
        github.com/gobuffalo/x v0.0.0-20190224155809-6bb134105960
        github.com/rs/cors v1.7.0
        github.com/unrolled/secure v0.0.0-20190103195806-76e6d4e9b90c
)
```
</details>

**Comment by Pulgafree** [comment]
Changing in actions/app.go from this:

`import(
...
"github.com/gobuffalo/buffalo-pop/pop/v2/popmw"
...
)`

to this:

`import(
...
"github.com/gobuffalo/buffalo-pop/v2/pop/popmw"
...
)`

Fixes the error.

**Comment by paganotoni** [comment]
This one is covered by the `fix` command. If you run `buffalo fix` after upgrading your buffalo binary it should do this replacement in your codebase.

**Comment by paganotoni** [comment]
Oh sorry! I misread the issue. Adding this to `v0.16.5`

**Comment by paganotoni** [comment]
Fixed in v0.16.5

**Comment by paganotoni** [comment]
Thanks for reporting this one @Pulgafree!
